runtime/criusupport/criusupport.cpp

```diff
@@ -481,7 +481,7 @@ Java_org_eclipse_openj9_criu_CRIUSupport_checkpointJVMImpl(JNIEnv *env,
 
 		toggleSuspendOnJavaThreads(currentThread, TRUE);
 
-		vm->extendedRuntimeFlags2 |= J9_EXTENDED_RUNTIME2_CRIU_SINGLE_THREAD_MODE;
+		vm->extendedRuntimeFlags2 |= J9_CHECKPOINT_STATE_SINGLE_THREAD_MODE;
```

Comment:
Should this be `vm->checkpointState.criuFlags |= J9_CHECKPOINT_STATE_SINGLE_THREAD_MODE;`?